src/admin/class-wordlift-admin-entity-list.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * Customization of the entity list in wp-admin/edit.php
+ * 
+ * @since 3.3.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * Wordlift_Entity_List_Service Class
+ *
+ * Handles the edit entities views.
+ */
+class Wordlift_Entity_List_Service {
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		
+	}
+
+	/**
+	 * Register custom columns for entity listing in backend
+	 * @see https://codex.wordpress.org/Plugin_API/Action_Reference/manage_posts_custom_column
+	 *
+	 * @since 3.2.0
+	 *
+	 * @param array $columns the current column.
```

Comment:
It's a separated service for all changes related to entity listing enhancement (so all the columns)